src/services/services.ts

```diff
@@ -1452,7 +1452,7 @@ export function updateLanguageServiceSourceFile(sourceFile: SourceFile, scriptSn
     return createLanguageServiceSourceFile(sourceFile.fileName, scriptSnapshot, options, version, /*setNodeParents*/ true, sourceFile.scriptKind);
 }
 
-const NoopCancellationToken: CancellationToken = {
+export const NoopCancellationToken: CancellationToken = {

```

Comment:
This also doesn't need to be exported anymore.

```suggestion
const NoopCancellationToken: CancellationToken = {
```

Reply:
It does. I need a cancellation token for `FindAllReferences.Core.getReferencedSymbolsForNode` and the one in the refactor context can be undefined.